v2/cards/bahia_bands.json

```diff
@@ -7,6 +7,8 @@
   "id": "bahia_bands",
   "influence_cost": 1,
   "is_unique": false,
+  "pronunciation_approx": "BYE-yah",
```

Comment:
any reason this doesn't have bands in here as well?

Reply:
a similar question for other cards.  Why does the pronunciation only have part of the title?

If that's the convention, cool cool, but i'm curious.

Reply:
This is more or less the convention AFAIK. I'm assuming English words with known pronunciation by native English speakers were omitted. They're not in the guide, and I don't trust myself to generate proper IPA for them on my own

Reply:
that's what I expected.  Thanks for confirming.  :)